glyphs-reader/src/font.rs

```diff
@@ -862,6 +891,51 @@ impl TryFrom<RawGlyph> for Glyph {
     }
 }
 
+impl RawFeature {
+    fn generate_name(&self) -> &'static str {
+        match self.automatic {
+            Some(1i64) => "automatic",
+            _ => "???",
+        }
+    }
+}
+
+// https://github.com/googlefonts/glyphsLib/blob/24b4d340e4c82948ba121dcfe563c1450a8e69c9/Lib/glyphsLib/builder/features.py#L90
+fn prefix_to_feature(prefix: RawFeature) -> Result<FeatureSnippet, Error> {
+    let name = match prefix.name.as_ref().map(|s| s.as_str()) {
+        Some("<anonymous>") => prefix.name.unwrap(),
+        Some(..) => prefix.name.unwrap(),
```

Comment:
what's the difference in practive between the two match branches above, namely `Some("<anonymous>")` and `Some(..)`, don't both return the same `prefix.name.unwrap()`?

Reply:
also, why not simply do

```rust
let name = match prefix.name {
    Some(name) => name,
    None => prefix.generate_name().to_string(),
}
```

Reply:
That's ... either a copy paste error or a series of code manipulations that left nonsensical code I just didn't notice. I think at some point <anonymous> did something else.